repositories.bzl

```diff
@@ -3,6 +3,13 @@ load("@bazel_gazelle//:deps.bzl", "go_repository")
 # bazel run //:gazelle -- update-repos -from_file=go.mod -to_macro=repositories.bzl%go_repositories
 
 def go_repositories():
+    go_repository(
+        name = "com_github_fsnotify_fsevents",
+        importpath = "github.com/fsnotify/fsevents",
+        sum = "h1:/125uxJvvoSDDBPen6yUZbil8J9ydKZnnl3TWWmvnkw=",
+        version = "v0.1.1",
+        patches = ["//:patches/fsnotify_fsevents_pr_38.diff"],
```

Comment:
Can we move this file to `third_party/fsnotify/fsevents/pr_38.diff`. `third_party` is, by convention where this sort of thing would usually go.